doc/source/running.rst

```diff
@@ -242,55 +164,51 @@ An example of the ``createUser`` command (must be run via the 'mongo' shell on a
    > db.createUser({user: "pbmAgent",
                     pwd: "securePassw0rd",
                     roles: [{db: "admin", role: "backup" },
+		            {db: "admin", role: "readWrite" },
                             {db: "admin", role: "clusterMonitor" },
                             { db: "admin", role: "restore" }],
                     authenticationRestrictions: [{ clientSource: ["127.0.0.1"]}]})
 
-Running |pbmctl| Commands
+Running |pbm.app| Commands
 --------------------------------------------------------------------------------
 
-|pbmctl| is the command line utility to control the backup system. Since
-n|pbmctl| needs to connect to the |bc|, you need to specify the |bc|
-`IP:Port`. The default values are `127.0.0.1:10001`. If you are running |pbmctl|
-from the same server where the coordinator is running, you can ommit the
-|opt-server-address| option.
+|pbm.app| is the command line utility to control the backup system. Before you
+can work with backups, make sure to set the remote store:
+
+.. code-block:: bash
+
+   $ pbm storage set store.yaml
```

Comment:
`--mongodb-uri` is needed here also (or it can be specified as env. variable), also `--config` is needed.
This would be an example of command:
`./pbm store set --config=storage-config.yaml --mongodb-uri="mongodb://localhost:27017/"`

Reply:
Correct.
Except we don't need the "./" in front the executable because it should be installed in their path.